specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/CommonDefinitions.json

```diff
@@ -2228,6 +2228,19 @@
      "description": "Family code of the resource SKU.",
      "type": "string"
     },
+    "computeMode": {
+     "description": "Shared or dedicated app hosting.",
+     "enum": [
+      "Shared",
+      "Dedicated",
+      "Dynamic"
+     ],
+     "type": "string",
+     "x-ms-enum": {
+      "name": "ComputeModeOptions",
```

Comment:
Looks like you are redefining the same enum `ComputeModeOptions` multiple times but with different values for `"modelAsString"`.  What you may do is
- Create a single definition for it and have it referenced wherever required
- Modify `"modelAsString":true` for each definition of `ComputeModeOptions`

I'd recommend the former

Reply:
yeah I was just trying to debug that 😄

I've gone with the latter for the moment to get the build to pass, and I'll now look into the former :)

Reply:
pushed a change for the former 🤞

Reply:
@dsgouda sorry to pester, but would you be able to advise/explain how I can fix this build on Java and Go, since it appears fine in the Node build?